package.json

```diff
@@ -47,11 +47,11 @@
     "typescript": "^5.4.5"
   },
   "engines": {
-    "pnpm": "^9.1.1",
+    "pnpm": "^9.1.2",
     "npm": "pnpm",
     "yarn": "pnpm",
     "node": "^20.12.1"
   },
   "private": true,
-  "packageManager": "pnpm@9.1.1"
+  "packageManager": "pnpm@9.1.2"
```

Comment:
doesn't deploy pipeline need to adapt to this as well?
Also i'd prefer keeping such upgrades outside of feature PR's like this one

Reply:
If I didn't make this change, CI would encounter an error that the pnpm version should be updated.

Reply:
[ok i saw this](https://github.com/interledger/testnet/pull/1333/files#diff-dd667e3260f679fc2c4cb8dd11a19dc1775d6a9e24025a99ef96c2775861c6dfR31)

Should still be a separate PR for that merged b4 this one, but ok